lib/model.js

```diff
@@ -1672,6 +1672,7 @@ class Model {
    * @param  {object}                                                    [options.having] Having options
    * @param  {string}                                                    [options.searchPath=DEFAULT] An optional parameter to specify the schema search_path (Postgres only)
    * @param  {boolean|Error}                                             [options.rejectOnEmpty=false] Throws an error when no records found
+   * @param  {boolean}                                                   [options.dotnotation] Allows including tables having the same attribute/column names - which have a dot in them. 
```

Comment:
I'm confused. Are we allowing dots in column names or that multiple tables have the same column name?

Reply:
I don't think we are talking about conflicting / overlapping properties but allowing dot notation in columns?

Reply:
> I'm confused. Are we allowing dots in column names or that multiple tables have the same column name?

I just realized the commit message is confusing, my bad

Sequelize does allow dot notation in columns 
but it raises errors when two tables have the same column name which includes dot in them, 
let's say 
`table1 (column1, "column2.label")` and `table2 (column1, "column2.label")`

then sequelize is generating a query like 
```sql
SELECT table1.column1,
       'column2.label',
       table2.column1,
       'column2.label'
FROM table1 AS "table1"
         LEFT OUTER JOIN table2 AS "table2" ON ....         
```
which raises ambiguity errors 
so having this `dotNotation` option will allow sequelize to concat the table_name even if a column includes a dot.